_data/simple-icons.json

```diff
@@ -1285,6 +1285,11 @@
             "hex": "1EABE2",
             "source": "https://cevo.com/"
         },
+        {
+            "title": "Chakra UI",
+            "hex": "319795",
+            "source": "https://github.com/chakra-ui/chakra-ui/blob/develop/logo/logomark-black.svg"
```

Comment:
Icon looks spot on, but could you update the source to [this page](https://github.com/chakra-ui/chakra-ui/blob/327e1624d22936abb43068e1f57054e43c9c6819/logo/logomark-colored.svg), just so we're linking to the specific version of the file?

Reply:
Done